tests/robot/DIRECTORY_TESTS/A.10_GET_VERSIONED_DIRECTORY/A.10__c._EHR_does_not_exist.robot

```diff
@@ -33,6 +33,11 @@ Alternative flow 2: get versioned directory from non-existent EHR
 
     create fake EHR
 
-    get DIRECTORY - fake ehr_id (JSON)
+        Log  TO CLARIFY WITH @PPAZOS: use `GET /ehr/ehr_id/directory` OR `GET /ehr/ehr_id/directory/\${version_uid}` here?
```

Comment:
@ppazos

Reply:
@testautomation /ehr/ehr_id/directory gets the current (lastest) version of EHR.directory, when version_uid is used is to get a specific, maybe previous version of the directory https://specifications.openehr.org/releases/ITS-REST/latest/ehr.html#directory-directory-get